_posts/[WIP]2016-08-02-2016-august-2016-couchbase-for-node.js

```diff
@@ -29,11 +29,18 @@ See you soon!
 
```

Comment:
The posts need to all be named: `_posts/[DATE]-[MEETUP_MONTH]-[MEETUP_DATE]-[SLUGS_TITLE].md`

here, there are a couple of things wrong:

1. `[WIP]` in the name
2. `.js` extension

That means that none of this will work -- when editing you should see a _rendered_ version of the Markdown option available.

Please retitle to: `2016-08-02-2016-august-2016-couchbase-for-node.md`